earth_moving/clutter/engine2.py

```diff
@@ -0,0 +1,346 @@
+# this file contains the PyBulletEnvironment class
```

Comment:
I accidentally added my previous comments on `engine.py` instead of `engine2.py`, but I think they are all relevant